.github/workflows/excavator.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     steps:
     - uses: actions/checkout@main
     - name: Excavate
+      if: github.repository == ‘ScoopInstaller/#xtras’
```

Comment:
Is `#xtras` a typo?

Reply:
Apostrophes (‘’) also looks weird. Normally should be `'ScoopInstaller/#xtras'`

Reply:
So why `#`...

Reply:
Typo, sorry.